### PR TITLE
spelling fixes, from Debian’s lintian tool

### PR DESCRIPTION
**Pull request by mirabilos**
There are more occurrences of “possibile” in the code, especially
in SMuFL glyph names and descriptions; these may be deliberate,
but someone should check this with SMuFL.

There are more spelling mistakes in the OpenSSL licence, but these
cannot be fixed in MuseScore.

### CODE REVIEW

**Comment by Jojo-Schmitz** [comment]
Like https://github.com/musescore/MuseScore/blob/master/fonts/smufl/glyphnames.json#L9608-L9614 and https://github.com/musescore/MuseScore/blob/master/libmscore/sym.cpp#L5038-L5039, These do look like typos to me, but indeed should get fixed upstream.

**Comment by lasconic** [comment]
It's called "overpressure possibile" in page 2 here https://issuu.com/louisgoldford/docs/memoire_involontaire
Could be a typo there too or the italian "possibile"?

**Comment by mirabilos** [comment]
>Could be a typo there too or the italian "possibile"?

Could be the Italian one, but the glyph name is English,
and even the description could conceivably use “possible”,
unless “possibile” is the name of a specific technique
and not translated in English scores, in which case
this would be correct to keep.

**Comment by IsaacWeiss** [comment]
I have seen "possibile" in sheet music, usually in an expressive instruction like "pesante possibile."

**Comment by Jojo-Schmitz** [comment]
`"possibile" alone I'd regard as the Italian spelling, but not in the context at hand here.

**Comment by Jojo-Schmitz** [comment]
See https://github.com/w3c/smufl/issues/74#issuecomment-320513315, 'possibile' is said to have the connotation of 'as much as possible', too suble for a non-native speaker like me, but seems to make sense, when comparing the glyphs string{Up,Down)Bow, stringsOverpresure{Up,Down}bow and stringsOberpresurePoissiblile{Up,Down}Bow, they do indicate an increase. See https://w3c.github.io/smufl/gitbook/tables/string-techniques.html.

File: mscore/importgtp-gp4.cpp [hunk 1]
```diff
@@ -192,7 +192,7 @@ bool GuitarPro4::readNote(int string, int staffIdx, Note* note)
       if (noteBits & 0x1) {               // note != beat
             int a = readUChar();          // length
             int b = readUChar();          // t
-            qDebug("          Time independend note len, len %d t %d", a, b);
+            qDebug("          time-independent note len, len %d t %d", a, b);
```
Comment (Jojo-Schmitz) [inline mscore/importgtp-gp4.cpp]: I'd drop the leading spaces and keep the upper-case T

File: mscore/importgtp.cpp [hunk 2]
```diff
@@ -1537,7 +1537,7 @@ void GuitarPro1::readNote(int string, Note* note)
       if (noteBits & 0x1) {               // note != beat
             int a = readUChar();          // length
             int b = readUChar();          // t
-            qDebug("Time independend note len, len %d t %d", a, b);
+            qDebug("time-independent note len, len %d t %d", a, b);
```
Comment (Jojo-Schmitz) [inline mscore/importgtp.cpp]: I'd Keep the upper-case T

File: mscore/shortcut.cpp [hunk 3]
```diff
@@ -3767,7 +3767,7 @@ void Shortcut::read(XmlReader& e)
                   for (const Shortcut& sc : _sc) {
                         for (const QKeySequence s : sc._keys) {
                               if (s == seq)
-                                    qDebug("ambigous shortcut for action <%s>", _key.data());
+                                    qDebug("ambiguous shortcut for action <%s>", _key.data());
```
Comment (Jojo-Schmitz) [inline mscore/shortcut.cpp]: Better upper case A?

**Comment by lasconic** [comment]
Manually merged in https://github.com/musescore/MuseScore/commit/9e275eb7ea8b1d078cc3573aa3eea0e59aca0e65

I removed the fixes in third party code and apply  the changes suggested by Jojo.

**Pull request closed without merging**